src/components/MultiAssetPreview/MultiAssetPreview.spec.tsx

```diff
@@ -0,0 +1,57 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import React from "react";
+import { MultiAssetPreview } from "./MultiAssetPreview";
+
+const MULTI_ASSET_PREVIEW_ID = "[data-test-id=multi-asset-preview]";
+
+describe("MultiAssetPreview Component", () => {
+    it("should render without images", () => {
+        mount(<MultiAssetPreview onClick={() => alert("Multi Asset clicked")} />);
+
+        cy.get(MULTI_ASSET_PREVIEW_ID).should("exist");
```

Comment:
When there are no images, you could additionally test if the `background-image` tag is set or not